test/lib/test_runner.go

```diff
@@ -48,7 +48,7 @@ import (
 const (
 	podLogsDir         = "pod-logs"
 	testPullSecretName = "kn-eventing-test-pull-secret"
-	MaxNamespaceSkip   = 50
+	MaxNamespaceSkip   = 100
```

Comment:
@mgencur any thoughts on this change?

Reply:
This change is good. We tested that in the PR knative-sandbox/eventing-kafka-broker#1227 . Increasing the value will help in cases with high parallelization and a lot of tests. I don't see any drawback in this change.

Reply:
Hi Matthias, Martin is ok with this change, see https://github.com/knative-sandbox/eventing-kafka-broker/pull/1227#issuecomment-915774256